package-lock.json

```diff
@@ -14,7 +14,7 @@
       "resolved": "https://registry.npmjs.org/@types/form-data/-/form-data-2.2.1.tgz",
       "integrity": "sha512-JAMFhOaHIciYVh8fb5/83nmuO/AHwmto+Hq7a9y8FzLDcC1KCU344XDOMEmahnrTFlHjgh4L0WJFczNIX2GxnQ==",
       "requires": {
-        "@types/node": "9.4.7"
+        "@types/node": "*"
```

Comment:
any reason for this file to change when the root package.json didn't change?

Reply:
I'm guessing updating to npm 6 changed the way lockfiles are generated?

Reply:
Probably, because I didn't change this. NPM must've done it.